src/CSSBattle/1_SimplySquare/shaokiat.html

```diff
@@ -0,0 +1,14 @@
+<div></div>
```

Comment:
could not see the code in the website as file name is not correct

Reply:
fea3899